.github/workflows/build.yml

```diff
@@ -24,22 +23,22 @@ jobs:
         os: [ubuntu-latest, windows-latest, macos-latest]
         node-version: [lts/*, 17.x, 16.x]
```

Comment:
check latest node versions
remove old odd versions

Reply:
5349c3f

Should remove 16.x or no?